cypress/features/regression/validations/checkboxValidations.feature

```diff
@@ -0,0 +1,51 @@
+Feature: Checkbox validations component
+  I want to change Checkbox validations component properties
+
+  Background: Open Checkbox validations component page
+    Given I open "Experimental Checkbox" component page validations in iframe
+
+  @positive
+  @validations
+  Scenario Outline: Verify the <state> validation of Checkbox in form component
+    Given I check "<position>" checkbox 2 times
+    When I hover mouse onto "<state>" icon into iFrame
```

Comment:
We could use more proper sentence `icon in iFrame`

Reply:
fixed